helm_deploy/templates/ingress.yaml

```diff
@@ -25,6 +25,7 @@ metadata:
     nginx.ingress.kubernetes.io/modsecurity-snippet: |
       SecRuleEngine On
       SecDefaultAction "phase:2,pass,log,tag:github_team=ministryofjustice/laa-crime-forms-team"
+      SecAction "id:900110,phase:1,nolog,pass,t:none,setvar:tx.inbound_anomaly_score_threshold=6"
```

Comment:
Not sure this is the way we should be doing this. Maybe instead we put this into detect only mode and see what comes up?

Reply:
Ok to go ahead with this but we should understand what it means.